lib/src/privtests.rs

```diff
@@ -196,6 +195,37 @@ fn verify_selinux_recurse(root: &Dir, path: &mut PathBuf, warn: bool) -> Result<
     Ok(())
 }
 
+#[context("Container tests")]
+fn test_build_lint(image: &str) -> Result<()> {
```

Comment:
Thanks for adding this test!  I'm working to clean up our testing framework a bit and you've hit some existing tech debt in that area.  I think we can get by with just unit tests to start.

Reply:
Ok, we have removed it.